vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/VGSCollect.kt

```diff
@@ -138,7 +138,9 @@ class VGSCollect {
     }
 
     private fun initializeCollect() {
-        client = ApiClient.newHttpClient()
+        client = ApiClient.newHttpClient().also {
+            it.getTemporaryStorage().setCustomHeaders(mapOf(generateAgentHeader(tracker.isEnabled)))
```

Comment:
Could we move this config to the ApiClient?

Reply:
@Endorf I have moved this code to a separate function, but I think it's better to update the agent header in `VGSCollect` to have only one place where it happens, I think in the future we most likely refactor headers logic.